source/_integrations/time_date.markdown

```diff
@@ -9,9 +9,11 @@ ha_release: pre 0.7
 ha_qa_scale: internal
 ---
 
-The time and date (`time_date`) sensor platform simple displays the time in various formats, the date, or both.
+The time and date (`time_date`) sensor platform adds one or more sensors to your Home Assistant state machine.
+It also displays the time in various formats, the date, or both in non-Lovelace mode.  
+Note that you have to add all options you need access to even if you don't want them to be displayed.
```

Comment:
I'm not sure what this means?

Reply:
See my previous comment.
If you want to access sensor.date, you need to add:
```
sensor:
    platform: time_date
    display_options:
       - date
```
even if you have no intention to see this date displayed or if you use HA in Lovelace mode.
That `display_options` is misleading and now means just `options` or `sensors`

Reply:
The definition of sensors/entities/integrations has nothing to do with displaying them. Those are 2 separate concerns. Hence my question, what this means.
Please remove it.

Reply:
> The definition of sensors/entities/integrations has nothing to do with displaying them. 

agree.
but what happens when you define
```
sensor:
   platform: time_date
```
?

> Those are 2 separate concerns. Hence my question, what this means.

from my point of view it means that currently to have a sensor that contains date I have to write
```
sensor:
    platform: time_date
    display_options:
       - date
```
instead of more intuitive
```
sensor:
    platform: time_date
    options:
       - date
```
exactly because "The definition of sensors/entities/integrations has nothing to do with displaying them", but in this integartion definition of a sensor tied to ```display_options``` variable, which is pretty misleading.

Reply:
That name could use an update, however, my comment remains the same.

Reply:
Does it mean that it's worth to have the wording as you requested (I think it's already is) and to create a new Feature Request (but there is no such thing for HA on Github afaik) referencing this PR as the cause?

Reply:
I think that that discussion is out of scope for this PR.

Reply:
I think it is directly related to this PR and I'm asking here because I know no other means to get information from the team behind HA (not because I just want to talk).

Reply:
The documentation should reflect the code base, how things are named in code is not a discussion for the documentation.

Reply:
Sure. Could you tell me _where_ I can discuss it with HA developers?

Reply:
Discord 👍